### PR TITLE
postgres source/destination - bump postgres driver version

### PR DESCRIPTION
**Pull request by evantahler**
Closes https://github.com/airbytehq/airbyte-internal-issues/issues/8249

### CODE REVIEW

**Review by stephane-airbyte — CHANGES_REQUESTED** [review]
you also should bump the version in deps.toml

**Review by stephane-airbyte — APPROVED** [review]

**Comment by evantahler** [comment]
Looks like `PostgresDestinationHandler` needs to implement `createNamespaces` - I need help with that

**Comment by evantahler** [comment]
New plan - the postgres destination(s) can't use the latest CDK until the Airbyte 1.0 work is complete (`ERROR: column "_airbyte_generation_id" of relation "users_final" does not exist`).  So, in this PR I'll bump the CDK dependencies, and when we update the postgres destination in general later this month, it will gain the new JDBC driver.

**Comment by evantahler** [comment]
@xiaohansong - any tips about what I should be doing here to get source-postgres to build with the latest CDK?

```
Error: [Task :airbyte-integrations:connectors:source-postgres:compileJava] constructor RelationalDbDebeziumPropertiesManager in class RelationalDbDebeziumPropertiesManager cannot be applied to given types;
    final var propertiesManager = new RelationalDbDebeziumPropertiesManager(
```

**Comment by theyueli** [comment]
> @xiaohansong - any tips about what I should be doing here to get source-postgres to build with the latest CDK?
> 
> ```
> Error: [Task :airbyte-integrations:connectors:source-postgres:compileJava] constructor RelationalDbDebeziumPropertiesManager in class RelationalDbDebeziumPropertiesManager cannot be applied to given types;
>     final var propertiesManager = new RelationalDbDebeziumPropertiesManager(
> ```

Is this actually related to @akashkulk 's change? The constructor now requires a new parameter:

https://github.com/airbytehq/airbyte/pull/39405/files#diff-3af961ccd56f952a3f466eb2f94fb6786785e37aa298fc1134e1aec2d2f81869R22

**Comment by xiaohansong** [comment]
@akashkulk has a fix in https://github.com/airbytehq/airbyte/pull/39474

**Comment by evantahler** [comment]
After rebasing the latest from @akashkulk, this PR is green!
Can I please get a review from @airbytehq/dbsources

**Review by xiaohansong — APPROVED** [review]

File: airbyte-integrations/connectors/source-postgres/build.gradle [hunk 1]
```diff
@@ -12,9 +12,9 @@ java {
 }
 
 airbyteJavaConnector {
-    cdkVersionRequired = '0.38.1'
+    cdkVersionRequired = '0.38.2'
     features = ['db-sources', 'datastore-postgres']
-    useLocalCdk = false
+    useLocalCdk = true
```
Comment (xiaohansong) [inline airbyte-integrations/connectors/source-postgres/build.gradle:17]: remember to change this back!

**Pull request merged**